x-pack/plugins/security_solution/public/detections/components/rules/pre_packaged_rules/update_callout.test.tsx

```diff
@@ -34,4 +59,82 @@ describe('UpdatePrePackagedRulesCallOut', () => {
 
     expect(wrapper.find('EuiCallOut')).toHaveLength(1);
   });
+
+  it('renders callOutMessage correctly: numberOfUpdatedRules > 0 and numberOfUpdatedTimelines = 0', () => {
+    shallow(
+      <UpdatePrePackagedRulesCallOut
+        loading={false}
+        numberOfUpdatedRules={1}
+        numberOfUpdatedTimelines={0}
+        updateRules={jest.fn()}
+      />
+    );
+
+    expect(spyUpdatePrepackagedRulesMsg).toHaveBeenCalledWith(1);
```

Comment:
What do you think about testing the text of the callout like

`expect(wrapper.find('data-test-subj["callout-message"]').text().includes(1)).toBeTruthy()`

instead of spying the `i18n` functions.